Bio/PDB/__init__.py

```diff
@@ -88,3 +102,67 @@
     from .SASA import ShrakeRupley
 except ImportError:
     pass
+
+_FormatToParser = {"pdb": PDBParser(), "cif": MMCIFParser()}
+
+
+def read(handle, format, id=None):
+    """Parse a file or handle using one of the available parsers to generate a structure.
+
+    Arguments:
+     - handle    - handle to the file, or the filename as a string
+     - format    - string describing the file format. can be [pdb, cif, mmtf]
+     - id        - optional id of the structure, if None header information or then base name is used
+
+    If you have the file name in a string 'filename', use:
+
+    >>> from Bio import PDB
+    >>> filename = "../Tests/PDB/1A8O.pdb"
+    >>> structure = PDB.read(filename, format="pdb")
+
+    The function also supports file handles and gzip files
+
+    >>> from Bio import PDB
+    >>> structure = PDB.read("../Tests/PDB/1A8O.pdb.gz", format="pdb")
+    """
+    # TODO *kwargs for the parser to allow QUIET or permissive mode
+
+    # Try and give helpful error messages:
+    if not isinstance(format, str):
+        raise TypeError("Need a string for the file format (lower case)")
+    if not format:
+        raise ValueError("Format required (lower case string)")
+    if format != format.lower():
+        raise ValueError("Format string '%s' should be lower case" % format)
+
+    # need to use different parser for mmtf versus pdb,cif
+    if format == "mmtf":
+        # TODO: Raise warning if ID is set. MMTFParser does not support passing an id
+        # MMTF needs a seperate parser as get_structure does only take 1 positional argument and does not support file handles
```

Comment:
I'd ignore `id` indeed. Alternatively, we can add a dummy id field to that `get_structure` so that we have a unified interface (easier than adding logic on this end).